build/teamcity/cockroach/nightlies/sqllogic_backup_restore_impl.sh

```diff
@@ -36,15 +35,8 @@ $BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci test -- --config=ci \
     --test_env=COCKROACH_LOGIC_TEST_BACKUP_RESTORE_PROBABILITY=0.5 \
     --test_env=GO_TEST_WRAP_TESTV=1 \
     --test_env=GO_TEST_WRAP=1 \
-    --test_env=GO_TEST_JSON_OUTPUT_FILE=$GO_TEST_JSON_OUTPUT_FILE.$config \
     --test_timeout=28800 \
     --test_env=GOOGLE_APPLICATION_CREDENTIALS="$GOOGLE_APPLICATION_CREDENTIALS" \
     || exit_status=$?
 
-process_test_json \
-  $BAZEL_BIN/pkg/cmd/testfilter/testfilter_/testfilter \
-  $BAZEL_BIN/pkg/cmd/github-post/github-post_/github-post \
```

Comment:
We can change line 9 
from:
`bazel build //pkg/cmd/bazci //pkg/cmd/github-post //pkg/cmd/testfilter --config=ci`
to:
`bazel build //pkg/cmd/bazci --config=ci`